src/components/Alert.ts

```diff
@@ -1,8 +1,8 @@
-import { DOM, StatelessComponent } from "react";
+import { StatelessComponent, createElement } from "react";
 
 export const Alert: StatelessComponent<{ message?: string }> = (props) =>
```

Comment:
Make alert generic so that it can be used for different kind of alert like success, info, etc.